src/Elders.Cronus.Projections.Cassandra/Infrastructure/TableRetention/TableRetentionOptions.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Elders.Cronus.Projections.Cassandra.Infrastructure
+{
+    public class TableRetentionOptions
+    {
+        public bool DeleteOldProjectionTables { get; set; } = false;
+        public uint NumberOfOldProjectionTablesToRetain { get; set; } = 2;
```

Comment:
If the option `NumberOfOldProjectionTablesToRetain` is 2 does this mean that there will be 3 tables in cassandra (excluding the snapshots)?

Reply:
`NumberOfOldProjectionTablesToRetain = 2` means there will be 2 old tables excluding the current one to a total of 3.